meta-lmp-base/recipes-kernel/jailhouse/jailhouse_git.bb

```diff
@@ -47,6 +47,17 @@ INMATES_DTB_DIR ?= "${JH_DATADIR}/inmates/dtb"
 # Inmate dtbs are manually installed (empty == all)
 JH_INMATE_DTB ?= ""
 
+EXTRA_OEMAKE += 'V=1'
+EXTRA_OEMAKE += 'PYTHON=python3'
+EXTRA_OEMAKE += 'LDFLAGS=""'
+EXTRA_OEMAKE += 'CC="${CC}"'
+EXTRA_OEMAKE += 'ARCH=${JH_ARCH}'
+EXTRA_OEMAKE += 'CROSS_COMPILE=${TARGET_PREFIX}'
+EXTRA_OEMAKE += 'KDIR=${STAGING_KERNEL_BUILDDIR}'
+EXTRA_OEMAKE += 'MODLIB="${D}${nonarch_base_libdir}/modules/${KERNEL_VERSION}"'
```

Comment:
This is an extra parameter, please add to the commit message why it was added.

Reply:
are you asking explicitly about the MODLIB parameter?

Reply:
Yes